easybuild/easyconfigs/p/PyTorch-extensions/PyTorch-extensions-2019.01-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-extensions'
+version = '2019.01'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rusty1s'
+description = """This is a bundle of extension libraries for PyTorch written by Matthias Fey."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+builddependencies = [('binutils', '2.31.1')]
+dependencies = [
+    ('Python', '3.7.2'),
+    ('PyTorch', '1.1.0', versionsuffix),
+    ('scikit-learn', '0.20.3'),
+    ('scikit-image', '0.15.0', versionsuffix),
+    ('numba', '0.43.1'),
+    ('h5py', '2.9.0'),
+    ('RDFlib', '4.2.2'),
+    ('google-drive-downloader', '0.4'),
+    ('plyfile', '0.7'),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('Torch-Scatter', '1.3.1', {
```

Comment:
You can cheat a bit with the extension names to avoid having to specify a custom `source_tmpl` for each: just use `torch_scatter` here; same below.

Reply:
Do you have a good reason for not using `torch-scatter` 1.3.2? If so, clarify it with a comment above this line. May be due to compatibility with PyTorch 1.1.0 which we're sort of stuck with in the `2019a` generation (unless we add the `PyTorch` version to `versionsuffix`, like I'm doing for PyTorch 1.4.0 + `foss/2019b` in an upcoming PR).

Same for older versions listed below, please clarify (see https://github.com/easybuilders/easybuild-easyconfigs/pull/9976/files#diff-8596d9a01c53dc3b4af88e0ae58b21b9R34)